example/app/views/layouts/embedded_app.html.erb

```diff
@@ -29,7 +29,7 @@
       });
     </script>
 
-    <%= javascript_include_tag 'application', "data-turbolinks-track" => true %>
+    <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
```

Comment:
why double quotes? I thought that standard was singe `'` unless you needed string interp. Why is rubocop suggesting different.